<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Hannah Holland-Moritz | selected publications</title>
  <meta name="description" content="Adventures in microbial ecology. 
">

  <link rel="shortcut icon" href="/assets/img/favicon.ico">

  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/publications/">
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        <a class="site-title" href="/"><strong>Hannah</strong> Holland-Moritz</a>
    </span>
    

    <nav class="site-nav">
      <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="/">about</a>

        <!-- Blog -->
        <a class="page-link" href="/blog/">blog</a>

        <!-- Pages -->
        
          
        
          
        
          
        
          
            <a class="page-link" href="/projects/">projects</a>
          
        
          
            <a class="page-link" href="/publications/">publications</a>
          
        
          
            <a class="page-link" href="/teaching/">teaching</a>
          
        
          
        

        <!-- CV link -->
        <a class="page-link" href="/assets/pdf/CVFall19.pdf">CV</a>

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">selected publications</h1>
    <h5 class="post-description">To see the most current list of publications, check out my <a href="https://scholar.google.com/citations?user=zETn5qAAAAAJ&hl=en" target="_blank">Google Scholar</a> page. For PDFs of papers published during my dissertation, see the Fierer Lab <a href="http://fiererlab.org/publications/" target="_blank">publications page</a>.</h5>
  </header>

  <article class="post-content selected publications clearfix">
    <h3 class="pubyear">In the Pipeline</h3>
<ol class="bibliography"><li>

<div id="Carini2020">
  
    <span class="title">The effects of spatial variability and relic DNA removal on the detection of temporal dynamics in soil microbial communities</span>
    <span class="author">
      
        
          
            
              
                Carini, Paul,
              
            
          
        
      
        
          
            
              
                Delgado-Baquerizo, Manuel,
              
            
          
        
      
        
          
            
              
                Hinckley, Eve-Lyn S.,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Brewer, Tess E,
              
            
          
        
      
        
          
            
              
                Rue, Garrett,
              
            
          
        
      
        
          
            
              
                Vanderburgh, Caihong,
              
            
          
        
      
        
          
            
              
                McKnight, Diane,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>accepted for publication at MBio</em>
    
    
      
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://www.biorxiv.org/content/early/2018/08/29/402438.full.pdf" target="_blank">pre-print</a>]
  
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Few studies have comprehensively investigated the temporal variability in soil microbial communities despite widespread recognition that the belowground environment is dynamic. In part, this stems from the challenges associated with the high degree of spatial heterogeneity in soil microbial communities1 and because the presence of relic DNA2 may mask temporal dynamics. Here we disentangle the relationships among spatial, temporal, and relic DNA effects on microbial communities in soils collected from contrasting hillslopes in Colorado, USA. These sites were chosen because they have distinct soil microbial communities and experience strong seasonal changes in precipitation and temperature regimes. We intensively sampled plots on each hillslope over one year to discriminate between temporal variability, the intra-plot spatial heterogeneity, and relic DNA effects on the soil prokaryotic and fungal communities. We show that the intra-plot spatial variability in microbial community composition was strong and independent of relic DNA effects and these spatial patterns persisted throughout the study. When controlling for intra-plot spatial variability, we identified significant temporal variability in both plots, particularly after relic DNA was removed, suggesting that relic DNA hinders the detection of important temporal dynamics in soil microbial communities. We also identified microbial taxa that exhibited shared temporal responses and we show that these responses were often predictable from temporal changes in soil conditions. These findings highlight approaches that can be used to better characterize temporal shifts in soil microbial communities, information that is critical for predicting the environmental preferences of individual soil microbial taxa and identifying linkages between soil microbial community composition and belowground dynamics.</p>
  </span>
  
</div>

</li></ol>

<h3 class="pubyear">Published</h3>

<h3 class="year">2019</h3>
<ol class="bibliography"><li>

<div id="ward2019evolutionary">
  
    <span class="title">Evolutionary Implications of Anoxygenic Phototrophy in the Bacterial Phylum <i>Candidatus</i> Eremiobacterota (WPS-2)</span>
    <span class="author">
      
        
          
            
              
                Ward, Lewis M.,
              
            
          
        
      
        
          
            
              
                Cardona, Tanai,
              
            
          
        
      
        
          
            
              and <em>Holland-Moritz, Hannah</em>
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Frontiers in Microbiology</em>
    
    
      2019
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.01658/full" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://drive.google.com/file/d/1TUjiKAoT5-Z-kGdx1195KDNMrE9wyCbT/view" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Genome-resolved environmental metagenomic sequencing has uncovered substantial previously unrecognized microbial diversity relevant for understanding the ecology and evolution of the biosphere, providing a more nuanced view of the distribution and ecological significance of traits including phototrophy across diverse niches. Recently, the capacity for bacteriochlorophyll-based anoxygenic photosynthesis has been proposed in the uncultured bacterial WPS-2 phylum (recently proposed as Candidatus Eremiobacterota) that are in close association with boreal moss. Here, we use phylogenomic analysis to investigate the diversity and evolution of phototrophic WPS-2. We demonstrate that phototrophic WPS-2 show significant genetic and metabolic divergence from other phototrophic and non-phototrophic lineages. The genomes of these organisms encode a new family of anoxygenic Type II photochemical reaction centers and other phototrophy-related proteins that are both phylogenetically and structurally distinct from those found in previously described phototrophs. We propose the name Candidatus Baltobacterales for the order-level aerobic WPS-2 clade which contains phototrophic lineages, from the Greek for “bog” or “swamp,” in reference to the typical habitat of phototrophic members of this clade.</p>
  </span>
  
</div>

</li></ol>

<h3 class="year">2018</h3>
<ol class="bibliography"><li>

<div id="holland2018novel">
  
    <span class="title">Novel bacterial lineages associated with boreal moss species</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Stuart, Julia,
              
            
          
        
      
        
          
            
              
                Lewis, Lily R,
              
            
          
        
      
        
          
            
              
                Miller, Samantha,
              
            
          
        
      
        
          
            
              
                Mack, Michelle C,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Environmental Microbiology</em>
    
    
      2018
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.14288" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="http://fiererlab.org/wp-content/uploads/2018/08/Holland-Moritz_etal_2018_EnvironMicro.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Mosses are critical components of boreal ecosystems where they typically account for a large proportion of net primary productivity and harbour diverse bacterial communities that can be the major source of biologically-fixed nitrogen in these ecosystems. Despite their ecological importance, we have limited understanding of how microbial communities vary across boreal moss species and the extent to which local site conditions may influence the composition of these bacterial communities. We used marker gene sequencing to analyze bacterial communities associated with seven boreal moss species collected near Fairbanks, AK, USA. We found that host identity was more important than site in determining bacterial community composition and that mosses harbour diverse lineages of potential N2 -fixers as well as an abundance of novel taxa assigned to understudied bacterial phyla (including candidate phylum WPS-2). We performed shotgun metagenomic sequencing to assemble genomes from the WPS-2 candidate phylum and found that these moss-associated bacteria are likely anoxygenic phototrophs capable of carbon fixation via RuBisCo with an ability to utilize byproducts of photorespiration from hosts via a glyoxylate shunt. These results give new insights into the metabolic capabilities of understudied bacterial lineages that associate with mosses and the importance of plant hosts in shaping their microbiomes.</p>
  </span>
  
</div>

</li></ol>

<h3 class="year">2017</h3>
<ol class="bibliography"><li>

<div id="stuart2017plant">
  
    <span class="title">Plant, Microbiome, and Biogeochemistry: Quantifying moss-associated N fixation in Alaska</span>
    <span class="author">
      
        
          
            
              
                Stuart, Julia,
              
            
          
        
      
        
          
            
              
                Mack, Michelle C,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, H</em>,
            
          
        
      
        
          
            
              
                Fierer, Noah,
              
            
          
        
      
        
          
            
              
                McDaniels, Stuart,
              
            
          
        
      
        
          
            
              
                and Lewis, Lily
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>In AGU Fall Meeting Abstracts</em>
    
    
      2017
    
    </span>
  

  <span class="links">
  
  
  
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
</div>

</li></ol>

<h3 class="year">2014</h3>
<ol class="bibliography"><li>

<div id="holland2014draft">
  
    <span class="title">Draft genome sequence of the pyridinediol-fermenting bacterium <i>Synergistes jonesii</i> 78-1</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah E</em>,
            
          
        
      
        
          
            
              
                Coil, David A,
              
            
          
        
      
        
          
            
              
                Badger, Jonathan H,
              
            
          
        
      
        
          
            
              
                Dmitrov, George I,
              
            
          
        
      
        
          
            
              
                Khouri, Hoda,
              
            
          
        
      
        
          
            
              
                Ward, Naomi L,
              
            
          
        
      
        
          
            
              
                Robb, Frank T,
              
            
          
        
      
        
          
            
              
                and Eisen, Jonathan A
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Announcements</em>
    
    
      2014
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mra.asm.org/content/2/4/e00833-14" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mra.asm.org/content/ga/2/4/e00833-14.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Here we present the draft genome of Synergistes jonesii 78-1, ATCC 49833, a member of the Synergistes phylum. This organism was isolated from the rumen of a Hawaiian goat and ferments pyridinediols. The assembly contains 2,747,397 bp in 61 contigs.</p>
  </span>
  
</div>

</li></ol>

<h3 class="year">2013</h3>
<ol class="bibliography"><li>

<div id="holland2013draft">
  
    <span class="title">Draft genome sequence of Leucobacter sp. strain UCD-THU (phylum <i>Actinobacteria</i>)</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah E</em>,
            
          
        
      
        
          
            
              
                Bevans, Dakota R,
              
            
          
        
      
        
          
            
              
                Lang, Jenna M,
              
            
          
        
      
        
          
            
              
                Darling, Aaron E,
              
            
          
        
      
        
          
            
              
                Eisen, Jonathan A,
              
            
          
        
      
        
          
            
              
                and Coil, David A
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Announcements</em>
    
    
      2013
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mra.asm.org/content/1/3/e00325-13" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mra.asm.org/content/ga/1/3/e00325-13.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Here we present the draft genome of Leucobacter sp. strain UCD-THU. The genome contains 3,317,267 bp in 11 scaffolds. This strain was isolated from a residential toilet as part of an undergraduate project to sequence reference genomes of microbes from the built environment.</p>
  </span>
  
</div>

</li></ol>


  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper">
    &copy; Copyright 2020 Hannah Holland-Moritz.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>. # Photos not taken by the author are from <a href="https://unsplash.com" target="_blank">Unsplash</a>.

    
  </div>

</footer>




    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


<!-- Load KaTeX -->
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.js"></script>
<script src="/assets/js/katex.js"></script>




<!-- Include custom icon fonts -->
<link rel="stylesheet" href="/assets/css/fontawesome-all.min.css">
<link rel="stylesheet" href="/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXXXXXX', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
