<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Hannah Holland-Moritz | publications</title>
  <meta name="description" content="Adventures in microbial ecology.
">

  <link rel="shortcut icon" href="/assets/img/favicon.ico">

  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/publications/">
  
  	







  

</head>

   <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">

  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        <a class="site-title" href="/"><strong>Hannah</strong> Holland-Moritz</a>
    </span>
    

    <nav class="site-nav">
      <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="/">about</a>
        <!-- Blog -->
        <a class="page-link" href="/blog/">blog</a>
        <!-- Pages -->
        
	  
            
	  
        
	  
        
	  
        
	  
            
	  
        
	  
            
	  
        
	  
            
              <a class="page-link" href="/projects/">projects</a>
            
	  
        
	  
            
              <a class="page-link" href="/publications/">publications</a>
            
	  
        
	  
            
              <a class="page-link" href="/teaching/">teaching</a>
            
	  
        
	  
            
	  
        
        <!-- CV link -->
        <a class="page-link" href="/assets/pdf/CVSummer23.pdf">CV</a>

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">publications</h1>
    <h5 class="post-description"><article>I try to keep this list as up to date as possible, but sometimes I fall behind. To see the most current list of publications, check out my <a href="https://scholar.google.com/citations?user=zETn5qAAAAAJ&hl=en" target="_blank">Google Scholar</a> page. For PDFs of papers published during my dissertation (2015-2021), see the Fierer Lab <a href="http://fiererlab.org/publications/" target="_blank">publications page</a>. If a pdf is not available on this page (this is usually because I am not the first author) feel free to <a href="mailto:hhollandmoritz@gmail.com" target="_blank">contact me</a> for a copy. </article></h5>
  </header>

  <article class="post-content publications clearfix">
    <h3 class="pubyear">In the pipeline</h3>
<p><em>papers submitted, in review and in revision</em></p>
<ol class="bibliography"><li>

<div id="Holland-Moritz2021">
  
    <span class="title">An ecological perspective on microbial genes of unknown function in soil</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Vanni, Chiara,
              
            
          
        
      
        
          
            
              
                Fernandez-Guerra, Antonio,
              
            
          
        
      
        
          
            
              
                Bissett, Andrew,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>bioRxiv</em>
    
    
      2021
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://link.springer.com/article/10.1007/s00442-021-05005-7" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://www.biorxiv.org/content/10.1101/2021.12.02.470747v1.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Genes that remain hypothetical, uncharacterized, and unannotated comprise a substantial portion of metagenomic datasets and are likely to be particularly prevalent in soils where poorly characterized taxa predominate. Documenting the prevalence, distribution, and potential roles of these genes of unknown function is an important first step to understanding their functional contributions in soil communities. We identified genes of unknown function from 50 soil metagenomes and analyzed their environmental distributions and ecological associations. We found that genes of unknown function are prevalent in soils, particularly fine-textured, higher pH soils that harbor greater abundances of Crenarchaeota, Gemmatimonadota, Nitrospirota, and Methylomirabilota. We identified 43 dominant (abundant and ubiquitous) gene clusters of unknown function and determined their associations with soil microbial phyla and other "known" genes. We found that these dominant unknown genes were commonly associated with microbial phyla that are relatively uncharacterized, with the majority of these dominant unknown genes associated with mobile genetic elements. This work demonstrates a strategy for investigating genes of unknown function in soils, emphasizes the biological insights that can be learned by adopting this strategy, and highlights specific hypotheses that warrant further investigation regarding the functional roles of abundant and ubiquitous genes of unknown function in soil metagenomes.Competing Interest StatementThe authors have declared no competing interest.</p>
  </span>
  
</div>

</li>
<li>

<div id="Rocci2023">
  
    <span class="title">Aligning theoretical and empirical representations of soil carbon-to-nitrogen stoichiometry with process-based terrestrial biogeochemistry models</span>
    <span class="author">
      
        
          
            
              
                Rocci, Katherine,
              
            
          
        
      
        
          
            
              
                Cleveland, Cory,
              
            
          
        
      
        
          
            
              
                Eastman, Brooke,
              
            
          
        
      
        
          
            
              
                Georgiou, Katerina,
              
            
          
        
      
        
          
            
              
                Grandy, A Stuart,
              
            
          
        
      
        
          
            
              
                Hartman, Melannie,
              
            
          
        
      
        
          
            
              
                Hauser, Emma,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Kyker-Snowman, Emily,
              
            
          
        
      
        
          
            
              
                Pierson, Derek,
              
            
          
        
      
        
          
            
              
                Reich, Peter,
              
            
          
        
      
        
          
            
              
                Schlerman, Else,
              
            
          
        
      
        
          
            
              
                and Wieder, William
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>In Review at Soil Biology and Biochemistry</em>
    
    
      2023
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Soil carbon-nitrogen stoichiometry (C:N) acts as a control over decomposition and soil organic matter formation and loss, making it a key soil property for understanding ecosystem dynamics. It is such a dominant feature of ecosystem function that soil C:N is parameterized in Earth System Models that are used to predict how ecosystems will behave under global environmental change. However, the controls of soil C:N and how they respond to increasing pressures from global change agents are not fully understood. The “foundational” controls on soil C:N, namely plant and microbial C:N, have been used to predict soil C:N, but fail to accurately simulate all ecosystems and may be insufficient for predictions under global environmental change. We present an “emerging” representation of controls of soil C:N that includes plant-microbe-mineral feedbacks that have been shown to regulate soil C:N. We argue that including representation of these emerging drivers in process-based terrestrial biogeochemistry models, which include biological N fixation, mycorrhizae, priming, root exudation of organic acids, and mineralogy, will improve mechanistic representation of soil C:N and associated processes. Such improvements will produce models that will better simulate a variety of ecological states and predict soil C:N when global changes modify plant-microbe-mineral interactions. Here, we align our empirical understanding of controls of soil C:N with those controls represented in models, and show that implementing emerging drivers of soil C:N results in different simulated outcomes at steady state and in response to elevated CO2. Our review and preliminary simulations support the need to incorporate emerging drivers of soil C:N into process-based terrestrial biogeochemistry models, allowing for both theoretical exploration of mechanisms and potentially more accurate predictions of land biogeochemical responses to global change.</p>
  </span>
  
</div>

</li>
<li>

<div id="Jay2023">
  
    <span class="title">Topographic Heterogeneity and Aspect Moderate Exposure to Climate Change Across an Alpine Tundra Hillslope</span>
    <span class="author">
      
        
          
            
              
                Jay, Katya,
              
            
          
        
      
        
          
            
              
                Wieder, William,
              
            
          
        
      
        
          
            
              
                Swenson, Sean,
              
            
          
        
      
        
          
            
              
                Knowles, John,
              
            
          
        
      
        
          
            
              
                Elmendorf, Sarah,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                and Suding, Katie
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>In Review at JGR Biogeosciences</em>
    
    
      2023
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Alpine tundra ecosystems are highly vulnerable to climate warming but are governed by local-scale abiotic heterogeneity, which makes it difficult to predict tundra responses to environmental change. Although land models are typically implemented at global scales, they can be applied at local scales to address process-based ecological questions. In this study, we ran ecosystem-scale Community Land Model (CLM) simulations with a novel hillslope hydrology configuration to represent topographically heterogeneous alpine tundra vegetation across a moisture gradient at Niwot Ridge, Colorado, USA. We used local observations to evaluate our simulations and investigated the role of topography and aspect in mediating patterns of snow, productivity, soil moisture, and soil temperature, as well as the potential exposure to climate change across an alpine tundra hillslope. Overall, our simulations captured observed gradients in abiotic conditions and productivity among heterogeneous, hydrologically connected vegetation communities (moist, wet, and dry). We found that south facing aspects were characterized by reduced snowpack and drier and warmer soils in all communities. When we extended our simulations to the year 2100, we found that earlier snowmelt altered the timing of runoff, with cascading effects on soil moisture, productivity, and growing season length. However, these effects were not distributed equally across the tundra, highlighting potential vulnerabilities of alpine vegetation in dry, wind-scoured, and south facing areas. Overall, our results demonstrate how land model outputs can be applied to advance process-based understanding of climate change impacts on ecosystem function.</p>
  </span>
  
</div>

</li>
<li>

<div id="Texeira2023">
  
    <span class="title">Conversion to managed land uses reduces soil carbon stocks and alters microbial community structure and function in a subtropical Atlantic Forest</span>
    <span class="author">
      
        
          
            
              
                Texeira, Cristhian,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Bayer, Cimélio,
              
            
          
        
      
        
          
            
              
                Eichelberger Granada, Camille,
              
            
          
        
      
        
          
            
              
                Sausen, Tanise Luisa,
              
            
          
        
      
        
          
            
              
                Tonial, Fabiana,
              
            
          
        
      
        
          
            
              
                Petry, Claudia,
              
            
          
        
      
        
          
            
              
                and Frey, Serita D.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>In Revision for Soil Biology and Biogeochemistry</em>
    
    
      2023
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Soil carbon (C) sequestration has been proposed as an alternative to mitigate the effects of climate change and to guarantee food safety. However, forest conversion to managed land uses has depleted soil C stocks and affected soil microbial community function and composition, potentially impacting a wide range of ecosystem services. Our objective was to investigate the effect of forest conversion to intensive land uses (pasture, cropland) on soil C dynamics and soil microbial community structure and function in the subtropical portion of the Atlantic Forest biome, one of the world’s most critical hotspots of biodiversity. For that, we evaluated four land uses - forest fragment, grassland, and agriculture with five or fifty years of cultivation. We analyzed soil C pools, edaphic variables, and soil bacterial/archaeal (16S) and fungal (ITS) communities. Forest conversion to anthropogenic land uses reduced soil C stocks and altered soil C dynamics. Bacterial/archaeal and fungal community composition was strongly correlated with pH, base saturation, and calcium (Ca) and magnesium (Mg) content. However, soil C pools were not associated with potential extracellular enzyme activity or microbial diversity. Finally, we identified microbes that were positively correlated with soil C stocks, potentially representing indicators of soil C storage. Our study indicates that land use change results in edaphic alterations with implications to soil C dynamics and microbial communities in the subtropical region of the Atlantic Forest biome.</p>
  </span>
  
</div>

</li></ol>
<h3 class="pubyear">Publications</h3>
<h4 class="pubyear"><i>Peer reviewed</i></h4>

<h4 class="year">2022</h4>
<ol class="bibliography"><li>

<div id="Fierer2022">
  
    <span class="title">A metagenomic investigation of spatial and temporal changes in sewage microbiomes across a university campus</span>
    <span class="author">
      
        
          
            
              
                Fierer, Noah,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Alexiev, Alexandra,
              
            
          
        
      
        
          
            
              
                Batther, Harpreet,
              
            
          
        
      
        
          
            
              
                Dragone, Nicholas B.,
              
            
          
        
      
        
          
            
              
                Friar, Liam,
              
            
          
        
      
        
          
            
              
                Gebert, Matthew J.,
              
            
          
        
      
        
          
            
              
                Gering, Sarah,
              
            
          
        
      
        
          
            
              
                Henley, Jessica B.,
              
            
          
        
      
        
          
            
              
                Jech, Sierra,
              
            
          
        
      
        
          
            
              
                Kibby, Emily M.,
              
            
          
        
      
        
          
            
              
                Melie, Tina,
              
            
          
        
      
        
          
            
              
                Patterson, William B.,
              
            
          
        
      
        
          
            
              
                Peterson, Eric,
              
            
          
        
      
        
          
            
              
                Schutz, Kyle,
              
            
          
        
      
        
          
            
              
                Stallard-Olivera, Elı́as,
              
            
          
        
      
        
          
            
              
                Sterrett, John,
              
            
          
        
      
        
          
            
              
                Walsh, Corinne,
              
            
          
        
      
        
          
            
              
                and Mansfeldt, Cresten
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>In press at mSystems</em>
    
    
      2022
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Wastewater microbial communities are not static and can vary significantly across time and space, but this variation and the factors driving the observed spatiotemporal variation often remain undetermined. We used a shotgun metagenomic approach to investigate changes in wastewater microbial communities across 17 locations in a sewer network, with samples collected from each location over a 3-week period. Fecal-derived bacteria constituted a relatively small fraction of the taxa found in the collected samples, highlighting the importance of environmental sources to the sewage microbiome. The prokaryotic communities were highly variable in composition depending on the location within the sampling network and this spatial variation was most strongly associated with location-specific differences in sewage pH. However, we also observed substantial temporal variation in the composition of the prokaryotic communities at individual locations. This temporal variation was asynchronous across sampling locations, emphasizing the importance of independently considering both spatial and temporal variation when assessing the wastewater microbiome. The spatiotemporal patterns in viral community composition closely tracked those of the prokaryotic communities, allowing us to putatively identify the bacterial hosts of some of the dominant viruses in these systems. Finally, we found that antibiotic resistance gene profiles also exhibit a high degree of spatiotemporal variability with most of these genes unlikely to be derived from fecal bacteria. Together these results emphasize the dynamic nature of the wastewater microbiome, the challenges associated with studying these systems, and the utility of metagenomic approaches for building a multi-faceted understanding of these microbial communities and their functional attributes.Importance Sewage systems harbor extensive microbial diversity, including microbes derived from both human and environmental sources. Studies of the sewage microbiome are useful for monitoring public health and the health of our infrastructure, but the sewage microbiome can be highly variable in ways that are often unresolved. We sequenced DNA recovered from wastewater samples collected over a 3-week period at 17 locations in a single sewer system to determine how these communities vary across time and space. Most of the wastewater bacteria, and the antibiotic resistance genes they harbor, were not derived from human feces, but human usage patterns did impact how the amounts and types of bacteria and bacterial genes we found in these systems varied over time. Likewise, the wastewater communities, including both bacteria and their viruses, varied depending on location within the sewage network, highlighting the challenges, and opportunities, in efforts to monitor and understand the sewage microbiome.</p>
  </span>
  
</div>

</li>
<li>

<div id="FAN2022">
  
    <span class="title">Identification of the rhizosphere microbes that actively consume plant-derived carbon</span>
    <span class="author">
      
        
          
            
              
                Fan, Kunkun,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Walsh, Corinne,
              
            
          
        
      
        
          
            
              
                Guo, Xisheng,
              
            
          
        
      
        
          
            
              
                Wang, Daozhong,
              
            
          
        
      
        
          
            
              
                Bai, Yang,
              
            
          
        
      
        
          
            
              
                Zhu, Yong-guan,
              
            
          
        
      
        
          
            
              
                Fierer, Noah,
              
            
          
        
      
        
          
            
              
                and Chu, Haiyan
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Soil Biology and Biochemistry</em>
    
    
      2022
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.sciencedirect.com/science/article/pii/S0038071722000347" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Rhizosphere microbial communities play essential roles in plant growth and health, with plant-derived carbon serving as the primary resource fueling the growth and activity of these root-associated communities. However, not all rhizosphere microbes are likely equivalent in their ability to metabolize root-derived carbon inputs, and far fewer studies have sought to identify the rhizosphere taxa, and the traits of those taxa that actively consume plant photosynthates. Here, we labeled wheat plants (Triticum aestivum L.) with 13C–CO2, combining stable isotope probing, quantitative PCR, marker gene sequencing, and shotgun metagenomic sequencing to identify rhizosphere microbes that metabolized plant-derived carbon and their genomic attributes. Those rhizosphere taxa that incorporated the plant-derived 13C were not necessarily the most abundant taxa in the rhizosphere. Rhizosphere microbes clearly differed in their capacity to consume plant-derived carbon, with the bacterial photosynthate consumers having distinct metabolic and genomic profiles with higher estimated potential growth rates and more genes associated with carbon metabolism, resource uptake, and potential for plant growth promotion. Together, this work highlights the important roles and the differential contributions of rhizosphere microbes to belowground carbon dynamics, building a more nuanced understanding of the complexity of plant-microbe interactions in the rhizosphere.</p>
  </span>
  
</div>

</li>
<li>

<div id="Dragone2022">
  
    <span class="title">Elevational Constraints on the Composition and Genomic Attributes of Microbial Communities in Antarctic Soils</span>
    <span class="author">
      
        
          
            
              
                Dragone, Nicholas B.,
              
            
          
        
      
        
          
            
              
                Henley, Jessica B.,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Diaz, Melisa,
              
            
          
        
      
        
          
            
              
                Hogg, Ian D.,
              
            
          
        
      
        
          
            
              
                Lyons, W. Berry,
              
            
          
        
      
        
          
            
              
                Wall, Diana H.,
              
            
          
        
      
        
          
            
              
                Adams, Byron J.,
              
            
          
        
      
        
          
            
              
                Fierer, Noah,
              
            
          
        
      
        
          
            
              
                and Mackelprang, Rachel
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>mSystems</em>
    
    
      2022
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://journals.asm.org/doi/10.1128/msystems.01330-21" target="_blank">HTML</a>]
  
  
    [<a href="/assets/pdf/https://journals.asm.org/doi/epub/10.1128/msystems.01330-21" target="_blank">PDF</a>]
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p> Antarctic soils represent an ideal system to study how environmental properties shape the taxonomic and functional diversity of microbial communities given the relatively low diversity of Antarctic soil microbial communities and the pronounced environmental gradients that occur across soils located in reasonable proximity to one another. Moreover, the challenging environmental conditions typical of most Antarctic soils present an opportunity to investigate the traits that allow soil microbes to persist in some of the most inhospitable habitats on Earth. The inland soils found on the Antarctic continent represent one of the more challenging environments for microbial life on Earth. Nevertheless, Antarctic soils harbor unique bacterial and archaeal (prokaryotic) communities able to cope with extremely cold and dry conditions. These communities are not homogeneous, and the taxonomic composition and functional capabilities (genomic attributes) of these communities across environmental gradients remain largely undetermined. We analyzed the prokaryotic communities in soil samples collected from across the Shackleton Glacier region of Antarctica by coupling quantitative PCR, marker gene amplicon sequencing, and shotgun metagenomic sequencing. We found that elevation was the dominant factor explaining differences in the structures of the soil prokaryotic communities, with the drier and saltier soils found at higher elevations harboring less diverse communities and unique assemblages of cooccurring taxa. The higher-elevation soil communities also had lower maximum potential growth rates (as inferred from metagenome-based estimates of codon usage bias) and an overrepresentation of genes associated with trace gas metabolism. Together, these results highlight the utility of assessing community shifts across pronounced environmental gradients to improve our understanding of the microbial diversity found in Antarctic soils and the strategies used by soil microbes to persist at the limits of habitability. IMPORTANCE Antarctic soils represent an ideal system to study how environmental properties shape the taxonomic and functional diversity of microbial communities given the relatively low diversity of Antarctic soil microbial communities and the pronounced environmental gradients that occur across soils located in reasonable proximity to one another. Moreover, the challenging environmental conditions typical of most Antarctic soils present an opportunity to investigate the traits that allow soil microbes to persist in some of the most inhospitable habitats on Earth. We used cultivation-independent methods to study the bacterial and archaeal communities found in soil samples collected from across the Shackleton Glacier region of the Transantarctic Mountains. We show that those environmental characteristics associated with elevation have the greatest impact on the structure of these microbial communities, with the colder, drier, and saltier soils found at higher elevations sustaining less diverse communities that were distinct from those in more hospitable soils with respect to their composition, genomic attributes, and overall life-history strategies. Notably, the harsher conditions found in higher-elevation soils likely select for taxa with lower maximum potential growth rates and an increased reliance on trace gas metabolism to support growth. </p>
  </span>
  
</div>

</li>
<li>

<div id="Vincent2021">
  
    <span class="title">Crossing Treeline: Bacterioplankton Communities of Alpine and Subalpine Rocky Mountain Lakes</span>
    <span class="author">
      
        
          
            
              
                Vincent, Kim,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Solon, Adam J.,
              
            
          
        
      
        
          
            
              
                Gendron, Eli M. S.,
              
            
          
        
      
        
          
            
              
                and Schmidt, Steven K.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Frontiers in Microbiology</em>
    
    
      2022
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.frontiersin.org/article/10.3389/fmicb.2021.533121" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>From the aboveground vegetation to the belowground microbes, terrestrial communities differ between the highly divergent alpine (above treeline) and subalpine (below treeline) ecosystems. Yet, much less is known about the partitioning of microbial communities between alpine and subalpine lakes. Our goal was to determine whether the composition of bacterioplankton communities of high-elevation mountain lakes differed across treeline, identify key players in driving the community composition, and identify potential environmental factors that may be driving differences. To do so, we compared bacterial community composition (using 16S rDNA sequencing) of alpine and subalpine lakes in the Southern Rocky Mountain ecoregion at two time points: once in the early summer and once in the late summer. In the early summer (July), shortly after peak runoff, bacterial communities of alpine lakes were distinct from subalpine lakes. Interestingly, by the end of the summer (approximately 5 weeks after the first visit in August), bacterial communities of alpine and subalpine lakes were no longer distinct. Several bacterial amplicon sequence variants (ASVs) were also identified as key players by significantly contributing to the community dissimilarity. The community divergence across treeline found in the early summer was correlated with several environmental factors, including dissolved organic carbon (DOC), pH, chlorophyll-a (chl-a), and total dissolved nitrogen (TDN). In this paper, we offer several potential scenarios driven by both biotic and abiotic factors that could lead to the observed patterns. While the mechanisms for these patterns are yet to be determined, the community dissimilarity in the early summer correlates with the timing of increased hydrologic connections with the terrestrial environment. Springtime snowmelt brings the flushing of mountain watersheds that connects terrestrial and aquatic ecosystems. This connectivity declines precipitously throughout the summer after snowmelt is complete. Regional climate change is predicted to bring alterations to precipitation and snowpack, which can modify the flushing of solutes, nutrients, and terrestrial microbes into lakes. Future preservation of the unique alpine lake ecosystem is dependent on a better understanding of ecosystem partitioning across treeline and careful consideration of terrestrial-aquatic connections in mountain watersheds.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2021</h4>
<ol class="bibliography"><li>

<div id="Stuart2021">
  
    <span class="title">The relationship of C and N stable isotopes to high-latitude moss-associated N<sub>2</sub> fixation</span>
    <span class="author">
      
        
          
            
              
                Stuart, Julia E M,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Jean, Mélanie,
              
            
          
        
      
        
          
            
              
                Miller, Samantha N,
              
            
          
        
      
        
          
            
              
                Ponciano, José Miguel,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                and Mack, Michelle C
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Oecologia</em>
    
    
      2021
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://link.springer.com/article/10.1007/s00442-021-05005-7" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Moss-associated N<sub>2</sub> fixation by epiphytic microbes is a key biogeochemical process in nutrient-limited high-latitude ecosystems. Abiotic drivers, such as temperature and moisture, and the identity of host mosses are critical sources of variation in N<sub>2</sub> fixation rates. An understanding of the potential interaction between these factors is essential for predicting N inputs as moss communities change with the climate. To further understand the drivers and results of N<sub>2</sub> fixation rate variation, we obtained natural abundance values of C and N isotopes and an associated rate of N<sub>2</sub> fixation with <sup>15</sup>N<sub>2</sub> gas incubations in 34 moss species collected in three regions across Alaska, USA. We hypothesized that &delta;<sup>15</sup>N values would increase toward 0‰ with higher N<sub>2</sub> fixation to reflect the increasing contribution of fixed N<sub>2</sub> in moss biomass. Second, we hypothesized that &delta;<sup>13</sup>C and N<sub>2</sub> fixation would be positively related, as enriched &delta;<sup>13</sup>C signatures reflect abiotic conditions favorable to N<sub>2</sub> fixation. We expected that the magnitude of these relationships would vary among types of host mosses, reflecting differences in anatomy and habitat. We found little support for our first hypothesis, with only a modest positive relationship between N<sub>2</sub> fixation rates and &delta;<sup>15</sup>N in a structural equation model. We found a significant positive relationship between &delta;<sup>13</sup>C and N<sub>2</sub> fixation only in Hypnales, where the probability of N<sub>2</sub> fixation activity reached 95% when &delta;<sup>13</sup>C values exceeded − 30.4‰. We conclude that moisture and temperature interact strongly with host moss identity in determining the extent to which abiotic conditions impact associated N<sub>2</sub> fixation rates.</p>
  </span>
  
</div>

</li>
<li>

<div id="Holland-Moritz2021">
  
    <span class="title">The Bacterial Communities of Alaskan Mosses and Their Contributions to N<sub>2</sub>-fixation</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Stuart, Julia E M,
              
            
          
        
      
        
          
            
              
                Lewis, Lily R,
              
            
          
        
      
        
          
            
              
                Miller, Samantha N,
              
            
          
        
      
        
          
            
              
                Mack, Michelle C,
              
            
          
        
      
        
          
            
              
                Ponciano, Jose Miguel,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Microbiome</em>
    
    
      2021
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01001-4" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://microbiomejournal.biomedcentral.com/track/pdf/10.1186/s40168-021-01001-4.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p><b>Background:</b> Mosses in high latitude ecosystems harbor diverse bacterial taxa, including N<sub>2</sub>-fixers which are key contributors to nitrogen dynamics in these systems. Yet, the relative importance of moss host species, and environmental factors, in structuring these microbial communities and their N<sub>2</sub>-fixing potential remains unclear. We studied 26 boreal and tundra moss species across 24 sites in Alaska, USA from 61° to 69° N. We used cultivation-independent approaches to characterize the variation in moss-associated bacterial communities as a function of host species identity and site characteristics. We also measured N<sub>2</sub>-fixation rates via <sup>15</sup>N<sub>2</sub> isotopic enrichment and identified potential N<sub>2</sub>-fixing bacteria using available literature and genomic information. <br /><b>Results:</b> Host species identity and host evolutionary history were both highly predictive of moss microbiome composition, highlighting strong phylogenetic coherence in these microbial communities. Although less important, light availability and temperature also influenced composition of the moss microbiome. Finally, we identified putative N<sub>2</sub>-fixing bacteria specific to some moss hosts, including potential N<sub>2</sub>-fixing bacteria outside well-studied cyanobacterial clades. <br /><b>Conclusions:</b> The strong effect of host identity on moss-associated bacterial communities demonstrates mosses’ utility for understanding plant-microbe interactions in non-leguminous systems. Our work also highlights the likely importance of novel bacterial taxa to N<sub>2</sub>-fixation in high-latitude ecosystems.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2020</h4>
<ol class="bibliography"><li>

<div id="stuart2020host">
  
    <span class="title">Host Identity as a Driver of Moss-Associated N<sub>2</sub> Fixation Rates in Alaska</span>
    <span class="author">
      
        
          
            
              
                Stuart, Julia EM,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Lewis, Lily R,
              
            
          
        
      
        
          
            
              
                Jean, Mélanie,
              
            
          
        
      
        
          
            
              
                Miller, Samantha N,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                Fierer, Noah,
              
            
          
        
      
        
          
            
              
                Ponciano, José Miguel,
              
            
          
        
      
        
          
            
              
                and Mack, Michelle C
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Ecosystems</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://link.springer.com/article/10.1007/s10021-020-00534-3" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Moss-associated N<sub>2</sub> fixation provides a substantial but heterogeneous input of new N to nutrient-limited ecosystems at high latitudes. In spite of the broad diversity of mosses found in boreal and Arctic ecosystems, the extent to which host moss identity drives variation in N<sub>2</sub> fixation rates remains largely undetermined. We used <sup>15</sup>N<sub>2</sub> incubations to quantify the fixation rates associated with 34 moss species from 24 sites ranging from 60° to 68° N in Alaska, USA. Remarkably, all sampled moss genera fixed N<sub>2</sub>, including well-studied feather and peat mosses and genera such as Tomentypnum, Dicranum, and Polytrichum. The total moss-associated N<sub>2</sub> fixation rates ranged from almost zero to 3.2 mg N m<sup>-2</sup> d<sup>-1</sup>, with an average of 0.8 mg N m<sup>-2</sup> d<sup>-1</sup>, based on abundance-weighted averages of all mosses summed for each site. Random forest models indicated that moss taxonomic family was a better predictor of rate variation across Alaska than any of the measured environmental factors, including site, pH, tree density, and mean annual precipitation and temperature. Consistent with this finding, mixed models showed that trends in N<sub>2</sub> fixation rates among moss genera were consistent across biomes. We also found “hotspots” of high fixation rates in one-fourth of sampled sites. Our results demonstrated the importance of moss identity in influencing N<sub>2</sub> fixation rates. This in turn indicates the potential utility of moss identity when making ecosystem N input predictions and exploring other sources of process rate variation.</p>
  </span>
  
</div>

</li>
<li>

<div id="Jean2020">
  
    <span class="title">Experimental assessment of tree canopy and leaf litter controls on the microbiome and nitrogen fixation rates of two boreal mosses</span>
    <span class="author">
      
        
          
            
              
                Jean, Mélanie,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Melvin, April M.,
              
            
          
        
      
        
          
            
              
                Johnstone, Jill F.,
              
            
          
        
      
        
          
            
              
                and Mack, Michelle C.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>New Phytologist</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.16611" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Summary Nitrogen (N<sub>2</sub>)-fixing moss microbial communities play key roles in nitrogen cycling of boreal forests. Forest type and leaf litter inputs regulate moss abundance, but how they control moss microbiomes and N<sub>2</sub>-fixation remains understudied. We examined impacts of forest type and broadleaf litter on microbial community composition and N<sub>2</sub>-fixation rates of <i>Hylocomium splendens</i> and <i>Pleurozium schreberi</i>. We conducted a moss transplant and leaf litter manipulation experiment at three sites with paired paper birch (<i>Betula neoalaskana</i>) and black spruce (<i>Picea mariana</i>) stands in Alaska. We characterized bacterial communities using marker gene sequencing, determined N<sub>2</sub>-fixation rates using stable isotopes (<sup>15</sup>N<sub>2</sub>), and measured environmental covariates. Mosses native to and transplanted into spruce stands supported generally higher N<sub>2</sub>-fixation and distinct microbial communities compared to similar treatments in birch stands. High leaf litter inputs shifted microbial community composition for both moss species and reduced N<sub>2</sub>-fixation rates for <i>H. splendens</i>, which had the highest rates. N<sub>2</sub>-fixation was positively associated with several bacterial taxa, including cyanobacteria. The moss microbiome and environmental conditions controlled N<sub>2</sub>-fixation at the stand- and transplant-scales. Predicted shifts from spruce- to deciduous-dominated stands will interact with the relative abundances of mosses supporting different microbiomes and N<sub>2</sub>-fixation rates, which could affect stand-level N inputs.</p>
  </span>
  
</div>

</li>
<li>

<div id="Carinie02776-19">
  
    <span class="title">Effects of Spatial Variability and Relic DNA Removal on the Detection of Temporal Dynamics in Soil Microbial Communities</span>
    <span class="author">
      
        
          
            
              
                Carini, Paul,
              
            
          
        
      
        
          
            
              
                Delgado-Baquerizo, Manuel,
              
            
          
        
      
        
          
            
              
                Hinckley, Eve-Lyn S.,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Brewer, Tess E.,
              
            
          
        
      
        
          
            
              
                Rue, Garrett,
              
            
          
        
      
        
          
            
              
                Vanderburgh, Caihong,
              
            
          
        
      
        
          
            
              
                McKnight, Diane,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>mBio</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mbio.asm.org/content/11/1/e02776-19.full" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mbio.asm.org/content/mbio/11/1/e02776-19.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Few studies have comprehensively investigated the temporal variability in soil microbial communities despite widespread recognition that the belowground environment is dynamic. In part, this stems from the challenges associated with the high degree of spatial heterogeneity in soil microbial communities and because the presence of relic DNA (DNA from dead cells or secreted extracellular DNA) may dampen temporal signals. Here, we disentangle the relationships among spatial, temporal, and relic DNA effects on prokaryotic and fungal communities in soils collected from contrasting hillslopes in Colorado, USA. We intensively sampled plots on each hillslope over 6 months to discriminate between temporal variability, intraplot spatial heterogeneity, and relic DNA effects on the soil prokaryotic and fungal communities. We show that the intraplot spatial variability in microbial community composition was strong and independent of relic DNA effects and that these spatial patterns persisted throughout the study. When controlling for intraplot spatial variability, we identified significant temporal variability in both plots over the 6-month study. These microbial communities were more dissimilar over time after relic DNA was removed, suggesting that relic DNA hinders the detection of important temporal dynamics in belowground microbial communities. We identified microbial taxa that exhibited shared temporal responses and show that these responses were often predictable from temporal changes in soil conditions. Our findings highlight approaches that can be used to better characterize temporal shifts in soil microbial communities, information that is critical for predicting the environmental preferences of individual soil microbial taxa and identifying linkages between soil microbial community composition and belowground processes.IMPORTANCE Nearly all microbial communities are dynamic in time. Understanding how temporal dynamics in microbial community structure affect soil biogeochemistry and fertility are key to being able to predict the responses of the soil microbiome to environmental perturbations. Here, we explain the effects of soil spatial structure and relic DNA on the determination of microbial community fluctuations over time. We found that intensive spatial sampling was required to identify temporal effects in microbial communities because of the high degree of spatial heterogeneity in soil and that DNA from nonliving sources masks important temporal patterns. We identified groups of microbes with shared temporal responses and show that these patterns were predictable from changes in soil characteristics. These results provide insight into the environmental preferences and temporal relationships between individual microbial taxa and highlight the importance of considering relic DNA when trying to detect temporal dynamics in belowground communities.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2019</h4>
<ol class="bibliography"><li>

<div id="ward2019evolutionary">
  
    <span class="title">Evolutionary Implications of Anoxygenic Phototrophy in the Bacterial Phylum <i>Candidatus</i> Eremiobacterota (WPS-2)</span>
    <span class="author">
      
        
          
            
              
                Ward, Lewis M.,
              
            
          
        
      
        
          
            
              
                Cardona, Tanai,
              
            
          
        
      
        
          
            
              and <em>Holland-Moritz, Hannah</em>
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Frontiers in Microbiology</em>
    
    
      2019
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.01658/full" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://drive.google.com/file/d/1TUjiKAoT5-Z-kGdx1195KDNMrE9wyCbT/view" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Genome-resolved environmental metagenomic sequencing has uncovered substantial previously unrecognized microbial diversity relevant for understanding the ecology and evolution of the biosphere, providing a more nuanced view of the distribution and ecological significance of traits including phototrophy across diverse niches. Recently, the capacity for bacteriochlorophyll-based anoxygenic photosynthesis has been proposed in the uncultured bacterial WPS-2 phylum (recently proposed as Candidatus Eremiobacterota) that are in close association with boreal moss. Here, we use phylogenomic analysis to investigate the diversity and evolution of phototrophic WPS-2. We demonstrate that phototrophic WPS-2 show significant genetic and metabolic divergence from other phototrophic and non-phototrophic lineages. The genomes of these organisms encode a new family of anoxygenic Type II photochemical reaction centers and other phototrophy-related proteins that are both phylogenetically and structurally distinct from those found in previously described phototrophs. We propose the name Candidatus Baltobacterales for the order-level aerobic WPS-2 clade which contains phototrophic lineages, from the Greek for “bog” or “swamp,” in reference to the typical habitat of phototrophic members of this clade.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2018</h4>
<ol class="bibliography"><li>

<div id="holland2018novel">
  
    <span class="title">Novel bacterial lineages associated with boreal moss species</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Stuart, Julia,
              
            
          
        
      
        
          
            
              
                Lewis, Lily R,
              
            
          
        
      
        
          
            
              
                Miller, Samantha,
              
            
          
        
      
        
          
            
              
                Mack, Michelle C,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Environmental Microbiology</em>
    
    
      2018
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.14288" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="http://fiererlab.org/wp-content/uploads/2018/08/Holland-Moritz_etal_2018_EnvironMicro.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Mosses are critical components of boreal ecosystems where they typically account for a large proportion of net primary productivity and harbour diverse bacterial communities that can be the major source of biologically-fixed nitrogen in these ecosystems. Despite their ecological importance, we have limited understanding of how microbial communities vary across boreal moss species and the extent to which local site conditions may influence the composition of these bacterial communities. We used marker gene sequencing to analyze bacterial communities associated with seven boreal moss species collected near Fairbanks, AK, USA. We found that host identity was more important than site in determining bacterial community composition and that mosses harbour diverse lineages of potential N<sub>2</sub> -fixers as well as an abundance of novel taxa assigned to understudied bacterial phyla (including candidate phylum WPS-2). We performed shotgun metagenomic sequencing to assemble genomes from the WPS-2 candidate phylum and found that these moss-associated bacteria are likely anoxygenic phototrophs capable of carbon fixation via RuBisCo with an ability to utilize byproducts of photorespiration from hosts via a glyoxylate shunt. These results give new insights into the metabolic capabilities of understudied bacterial lineages that associate with mosses and the importance of plant hosts in shaping their microbiomes.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2014</h4>
<ol class="bibliography"><li>

<div id="GUNAWARDANA20141">
  
    <span class="title">Isolation of PCR quality microbial community DNA from heavily contaminated environments</span>
    <span class="author">
      
        
          
            
              
                Gunawardana, Manjula,
              
            
          
        
      
        
          
            
              
                Chang, Simon,
              
            
          
        
      
        
          
            
              
                Jimenez, Abraham,
              
            
          
        
      
        
          
            
              
                Holland-Moritz, Daniel,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                La Val, Taylor P.,
              
            
          
        
      
        
          
            
              
                Lund, Craig,
              
            
          
        
      
        
          
            
              
                Mullen, Madeline,
              
            
          
        
      
        
          
            
              
                Olsen, John,
              
            
          
        
      
        
          
            
              
                Sztain, Terra A.,
              
            
          
        
      
        
          
            
              
                Yoo, Jennifer,
              
            
          
        
      
        
          
            
              
                Moss, John A.,
              
            
          
        
      
        
          
            
              
                and Baum, Marc M.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Journal of Microbiological Methods</em>
    
    
      2014
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.sciencedirect.com/science/article/pii/S0167701214001055?via%3Dihub" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Asphalts, biochemically degraded oil, contain persistent, water-soluble compounds that pose a significant challenge to the isolation of PCR quality DNA. The adaptation of existing DNA purification protocols and commercial kits proved unsuccessful at overcoming this hurdle. Treatment of aqueous asphalt extracts with a polyamide resin afforded genomic microbial DNA templates that could readily be amplified by PCR. Physicochemically distinct asphalt samples from five natural oil seeps successfully generated the expected 291bp amplicons targeting a region of the 16S rRNA gene, illustrating the robustness of the method. DNA recovery yields were in the 50–80% range depending on how the asphalt sample was seeded with exogenous DNA. The scope of the new method was expanded to include soil with high humic acid content. DNA from soil samples spiked with a range of humic acid concentrations was extracted with a commercial kit followed by treatment with the polyamide resin. The additional step significantly improved the purity of the DNA templates, especially at high humic acid concentrations, based on qPCR analysis of the bacterial 16S rRNA genes. The new method has the advantages of being inexpensive, simple, and rapid and should provide a valuable addition to protocols in the field of petroleum and soil microbiology.</p>
  </span>
  
</div>

</li></ol>

<h4 class="pubyear"><i>Not peer reviewed</i></h4>

<h4 class="year">2014</h4>
<ol class="bibliography"><li>

<div id="holland2014draft">
  
    <span class="title">Draft genome sequence of the pyridinediol-fermenting bacterium <i>Synergistes jonesii</i> 78-1</span>
    <span class="author">
      
        
          
            
              
                Holland-Moritz, Hannah E,
              
            
          
        
      
        
          
            
              
                Coil, David A,
              
            
          
        
      
        
          
            
              
                Badger, Jonathan H,
              
            
          
        
      
        
          
            
              
                Dmitrov, George I,
              
            
          
        
      
        
          
            
              
                Khouri, Hoda,
              
            
          
        
      
        
          
            
              
                Ward, Naomi L,
              
            
          
        
      
        
          
            
              
                Robb, Frank T,
              
            
          
        
      
        
          
            
              
                and Eisen, Jonathan A
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Announcements</em>
    
    
      2014
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mra.asm.org/content/2/4/e00833-14" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mra.asm.org/content/ga/2/4/e00833-14.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Here we present the draft genome of Synergistes jonesii 78-1, ATCC 49833, a member of the Synergistes phylum. This organism was isolated from the rumen of a Hawaiian goat and ferments pyridinediols. The assembly contains 2,747,397 bp in 61 contigs.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2013</h4>
<ol class="bibliography"><li>

<div id="holland2013draft">
  
    <span class="title">Draft genome sequence of Leucobacter sp. strain UCD-THU (phylum <i>Actinobacteria</i>)</span>
    <span class="author">
      
        
          
            
              
                Holland-Moritz, Hannah E,
              
            
          
        
      
        
          
            
              
                Bevans, Dakota R,
              
            
          
        
      
        
          
            
              
                Lang, Jenna M,
              
            
          
        
      
        
          
            
              
                Darling, Aaron E,
              
            
          
        
      
        
          
            
              
                Eisen, Jonathan A,
              
            
          
        
      
        
          
            
              
                and Coil, David A
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Announcements</em>
    
    
      2013
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mra.asm.org/content/1/3/e00325-13" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mra.asm.org/content/ga/1/3/e00325-13.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Here we present the draft genome of Leucobacter sp. strain UCD-THU. The genome contains 3,317,267 bp in 11 scaffolds. This strain was isolated from a residential toilet as part of an undergraduate project to sequence reference genomes of microbes from the built environment.</p>
  </span>
  
</div>

</li></ol>


  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper">
    &copy; Copyright 2023 Hannah Holland-Moritz.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>. </br> Photos not taken by the author or otherwise credited are from <a href="https://unsplash.com" target="_blank">Unsplash</a>. </br>

    
        Page updated: 2023-09-12.
    
  </div>

</footer>




    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


<!-- Load KaTeX -->
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.js"></script>
<script src="/assets/js/katex.js"></script>




<!-- Include custom icon fonts -->
<link rel="stylesheet" href="/assets/css/fontawesome-all.min.css">
<link rel="stylesheet" href="/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXXXXXX', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
