<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Hannah Holland-Moritz | selected publications</title>
  <meta name="description" content="Adventures in microbial ecology. 
">

  <link rel="shortcut icon" href="/assets/img/favicon.ico">

  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/publications/">
  
  	







  

</head>

   <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">

  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        <a class="site-title" href="/"><strong>Hannah</strong> Holland-Moritz</a>
    </span>
    

    <nav class="site-nav">
      <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="/">about</a>
        <!-- Blog -->
        <a class="page-link" href="/blog/">blog</a>
        <!-- Pages -->
        
	  
            
	  
        
	  
        
	  
        
	  
            
	  
        
	  
            
	  
        
	  
            
              <a class="page-link" href="/projects/">projects</a>
            
	  
        
	  
            
              <a class="page-link" href="/publications/">publications</a>
            
	  
        
	  
            
              <a class="page-link" href="/teaching/">teaching</a>
            
	  
        
	  
            
	  
        
        <!-- CV link -->
        <a class="page-link" href="/assets/pdf/CVSpring20.pdf">CV</a>

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">selected publications</h1>
    <h5 class="post-description"><article>To see the most current list of publications, check out my <a href="https://scholar.google.com/citations?user=zETn5qAAAAAJ&hl=en" target="_blank">Google Scholar</a> page. For PDFs of papers published during my dissertation, see the Fierer Lab <a href="http://fiererlab.org/publications/" target="_blank">publications page</a>. If a pdf is not available on this page feel free to <a href="mailto:hhollandmoritz@gmail.com" target="_blank">contact me</a> for a copy. </article></h5>
  </header>

  <article class="post-content selected publications clearfix">
    <h3 class="pubyear">In the pipeline</h3>
<ol class="bibliography"><li>

<div id="Holland-Moritz2021">
  
    <span class="title">The bacterial communities of Alaskan mosses and their contributions to N<sub>2</sub>-fixation</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Stuart, Julia E. M.,
              
            
          
        
      
        
          
            
              
                Lewis, Lily R.,
              
            
          
        
      
        
          
            
              
                Miller, Samantha N.,
              
            
          
        
      
        
          
            
              
                Mack, Michelle C.,
              
            
          
        
      
        
          
            
              
                Ponciano, José Miguel,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F.,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>In press at Microbiome</em>
    
    
      <i>2021</i>
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.researchsquare.com/article/rs-80075/v1" target="_blank">HTML</a>]
  
  
    [<a href="/assets/pdf/https://assets.researchsquare.com/files/rs-80075/v1_stamped.pdf" target="_blank">PDF</a>]
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p><p><strong>Background:</strong>
Mosses in high latitude ecosystems harbor diverse bacterial taxa, including N<sub>2</sub>-fixers which are key contributors to nitrogen dynamics in these systems. Yet, the relative importance of moss host species, and environmental factors, in structuring these microbial communities and their N<sub>2</sub>-fixing potential remains unclear. We studied 26 boreal and tundra moss species across 24 sites in Alaska, USA from 61º to 69º N. We used cultivation-independent approaches to characterize the variation in moss-associated bacterial communities as a function of host species identity and site characteristics. We also measured N<sub>2</sub>-fixation rates via <sup>15</sup>N<sub>2</sub> isotopic enrichment and identified potential N<sub>2</sub>-fixing bacteria using available literature and genomic information.</p>

<p><strong>Results:</strong>
Host species identity and host evolutionary history were both highly predictive of moss microbiome composition, highlighting strong phylogenetic coherence in these microbial communities. Although less important, light availability and temperature also influenced composition of the moss microbiome. Finally, we identified putative N<sub>2</sub>-fixing bacteria specific to some moss hosts, including potential N<sub>2</sub>-fixing bacteria outside well-studied cyanobacterial clades.</p>
<p><strong>Conclusions:</strong>
The strong effect of host identity on moss-associated bacterial communities demonstrates mosses’ utility for understanding plant-microbe interactions in non-leguminous systems. Our work also highlights the likely importance of novel bacterial taxa to N<sub>2</sub>-fixation in high-latitude ecosystems.</p></p>
  </span>
  
</div>

</li></ol>

<h3 class="pubyear">Publications</h3>
<h4 class="pubyear"><i>Peer reviewed</i></h4>

<h4 class="year">2021</h4>
<ol class="bibliography"></ol>

<h4 class="year">2020</h4>
<ol class="bibliography"><li>

<div id="stuart2020host">
  
    <span class="title">Host Identity as a Driver of Moss-Associated N 2 Fixation Rates in Alaska</span>
    <span class="author">
      
        
          
            
              
                Stuart, Julia EM,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Lewis, Lily R,
              
            
          
        
      
        
          
            
              
                Jean, Mélanie,
              
            
          
        
      
        
          
            
              
                Miller, Samantha N,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                Fierer, Noah,
              
            
          
        
      
        
          
            
              
                Ponciano, José Miguel,
              
            
          
        
      
        
          
            
              
                and Mack, Michelle C
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Ecosystems</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
  
  
  
    [<a href="https://link.springer.com/article/10.1007/s10021-020-00534-3" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
</div>

</li>
<li>

<div id="Jean2020">
  
    <span class="title">Experimental assessment of tree canopy and leaf litter controls on the microbiome and nitrogen fixation rates of two boreal mosses</span>
    <span class="author">
      
        
          
            
              
                Jean, Mélanie,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Melvin, April M.,
              
            
          
        
      
        
          
            
              
                Johnstone, Jill F.,
              
            
          
        
      
        
          
            
              
                and Mack, Michelle C.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>New Phytologist</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.16611" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Summary Nitrogen (N<sub>2</sub>)-fixing moss microbial communities play key roles in nitrogen cycling of boreal forests. Forest type and leaf litter inputs regulate moss abundance, but how they control moss microbiomes and N<sub>2</sub>-fixation remains understudied. We examined impacts of forest type and broadleaf litter on microbial community composition and N<sub>2</sub>-fixation rates of <i>Hylocomium splendens</i> and <i>Pleurozium schreberi</i>. We conducted a moss transplant and leaf litter manipulation experiment at three sites with paired paper birch (<i>Betula neoalaskana</i>) and black spruce (<i>Picea mariana</i>) stands in Alaska. We characterized bacterial communities using marker gene sequencing, determined N<sub>2</sub>-fixation rates using stable isotopes (<sup>15</sup>N<sub>2</sub>), and measured environmental covariates. Mosses native to and transplanted into spruce stands supported generally higher N<sub>2</sub>-fixation and distinct microbial communities compared to similar treatments in birch stands. High leaf litter inputs shifted microbial community composition for both moss species and reduced N<sub>2</sub>-fixation rates for <i>H. splendens</i>, which had the highest rates. N<sub>2</sub>-fixation was positively associated with several bacterial taxa, including cyanobacteria. The moss microbiome and environmental conditions controlled N<sub>2</sub>-fixation at the stand- and transplant-scales. Predicted shifts from spruce- to deciduous-dominated stands will interact with the relative abundances of mosses supporting different microbiomes and N<sub>2</sub>-fixation rates, which could affect stand-level N inputs.</p>
  </span>
  
</div>

</li>
<li>

<div id="Carinie02776-19">
  
    <span class="title">Effects of Spatial Variability and Relic DNA Removal on the Detection of Temporal Dynamics in Soil Microbial Communities</span>
    <span class="author">
      
        
          
            
              
                Carini, Paul,
              
            
          
        
      
        
          
            
              
                Delgado-Baquerizo, Manuel,
              
            
          
        
      
        
          
            
              
                Hinckley, Eve-Lyn S.,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Brewer, Tess E.,
              
            
          
        
      
        
          
            
              
                Rue, Garrett,
              
            
          
        
      
        
          
            
              
                Vanderburgh, Caihong,
              
            
          
        
      
        
          
            
              
                McKnight, Diane,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>mBio</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mbio.asm.org/content/11/1/e02776-19.full" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mbio.asm.org/content/mbio/11/1/e02776-19.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Few studies have comprehensively investigated the temporal variability in soil microbial communities despite widespread recognition that the belowground environment is dynamic. In part, this stems from the challenges associated with the high degree of spatial heterogeneity in soil microbial communities and because the presence of relic DNA (DNA from dead cells or secreted extracellular DNA) may dampen temporal signals. Here, we disentangle the relationships among spatial, temporal, and relic DNA effects on prokaryotic and fungal communities in soils collected from contrasting hillslopes in Colorado, USA. We intensively sampled plots on each hillslope over 6 months to discriminate between temporal variability, intraplot spatial heterogeneity, and relic DNA effects on the soil prokaryotic and fungal communities. We show that the intraplot spatial variability in microbial community composition was strong and independent of relic DNA effects and that these spatial patterns persisted throughout the study. When controlling for intraplot spatial variability, we identified significant temporal variability in both plots over the 6-month study. These microbial communities were more dissimilar over time after relic DNA was removed, suggesting that relic DNA hinders the detection of important temporal dynamics in belowground microbial communities. We identified microbial taxa that exhibited shared temporal responses and show that these responses were often predictable from temporal changes in soil conditions. Our findings highlight approaches that can be used to better characterize temporal shifts in soil microbial communities, information that is critical for predicting the environmental preferences of individual soil microbial taxa and identifying linkages between soil microbial community composition and belowground processes.IMPORTANCE Nearly all microbial communities are dynamic in time. Understanding how temporal dynamics in microbial community structure affect soil biogeochemistry and fertility are key to being able to predict the responses of the soil microbiome to environmental perturbations. Here, we explain the effects of soil spatial structure and relic DNA on the determination of microbial community fluctuations over time. We found that intensive spatial sampling was required to identify temporal effects in microbial communities because of the high degree of spatial heterogeneity in soil and that DNA from nonliving sources masks important temporal patterns. We identified groups of microbes with shared temporal responses and show that these patterns were predictable from changes in soil characteristics. These results provide insight into the environmental preferences and temporal relationships between individual microbial taxa and highlight the importance of considering relic DNA when trying to detect temporal dynamics in belowground communities.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2019</h4>
<ol class="bibliography"><li>

<div id="ward2019evolutionary">
  
    <span class="title">Evolutionary Implications of Anoxygenic Phototrophy in the Bacterial Phylum <i>Candidatus</i> Eremiobacterota (WPS-2)</span>
    <span class="author">
      
        
          
            
              
                Ward, Lewis M.,
              
            
          
        
      
        
          
            
              
                Cardona, Tanai,
              
            
          
        
      
        
          
            
              and <em>Holland-Moritz, Hannah</em>
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Frontiers in Microbiology</em>
    
    
      2019
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.01658/full" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://drive.google.com/file/d/1TUjiKAoT5-Z-kGdx1195KDNMrE9wyCbT/view" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Genome-resolved environmental metagenomic sequencing has uncovered substantial previously unrecognized microbial diversity relevant for understanding the ecology and evolution of the biosphere, providing a more nuanced view of the distribution and ecological significance of traits including phototrophy across diverse niches. Recently, the capacity for bacteriochlorophyll-based anoxygenic photosynthesis has been proposed in the uncultured bacterial WPS-2 phylum (recently proposed as Candidatus Eremiobacterota) that are in close association with boreal moss. Here, we use phylogenomic analysis to investigate the diversity and evolution of phototrophic WPS-2. We demonstrate that phototrophic WPS-2 show significant genetic and metabolic divergence from other phototrophic and non-phototrophic lineages. The genomes of these organisms encode a new family of anoxygenic Type II photochemical reaction centers and other phototrophy-related proteins that are both phylogenetically and structurally distinct from those found in previously described phototrophs. We propose the name Candidatus Baltobacterales for the order-level aerobic WPS-2 clade which contains phototrophic lineages, from the Greek for “bog” or “swamp,” in reference to the typical habitat of phototrophic members of this clade.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2018</h4>
<ol class="bibliography"><li>

<div id="holland2018novel">
  
    <span class="title">Novel bacterial lineages associated with boreal moss species</span>
    <span class="author">
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                Stuart, Julia,
              
            
          
        
      
        
          
            
              
                Lewis, Lily R,
              
            
          
        
      
        
          
            
              
                Miller, Samantha,
              
            
          
        
      
        
          
            
              
                Mack, Michelle C,
              
            
          
        
      
        
          
            
              
                McDaniel, Stuart F,
              
            
          
        
      
        
          
            
              
                and Fierer, Noah
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Environmental Microbiology</em>
    
    
      2018
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.14288" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="http://fiererlab.org/wp-content/uploads/2018/08/Holland-Moritz_etal_2018_EnvironMicro.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Mosses are critical components of boreal ecosystems where they typically account for a large proportion of net primary productivity and harbour diverse bacterial communities that can be the major source of biologically-fixed nitrogen in these ecosystems. Despite their ecological importance, we have limited understanding of how microbial communities vary across boreal moss species and the extent to which local site conditions may influence the composition of these bacterial communities. We used marker gene sequencing to analyze bacterial communities associated with seven boreal moss species collected near Fairbanks, AK, USA. We found that host identity was more important than site in determining bacterial community composition and that mosses harbour diverse lineages of potential N2 -fixers as well as an abundance of novel taxa assigned to understudied bacterial phyla (including candidate phylum WPS-2). We performed shotgun metagenomic sequencing to assemble genomes from the WPS-2 candidate phylum and found that these moss-associated bacteria are likely anoxygenic phototrophs capable of carbon fixation via RuBisCo with an ability to utilize byproducts of photorespiration from hosts via a glyoxylate shunt. These results give new insights into the metabolic capabilities of understudied bacterial lineages that associate with mosses and the importance of plant hosts in shaping their microbiomes.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2014</h4>
<ol class="bibliography"><li>

<div id="GUNAWARDANA20141">
  
    <span class="title">Isolation of PCR quality microbial community DNA from heavily contaminated environments</span>
    <span class="author">
      
        
          
            
              
                Gunawardana, Manjula,
              
            
          
        
      
        
          
            
              
                Chang, Simon,
              
            
          
        
      
        
          
            
              
                Jimenez, Abraham,
              
            
          
        
      
        
          
            
              
                Holland-Moritz, Daniel,
              
            
          
        
      
        
          
            
              <em>Holland-Moritz, Hannah</em>,
            
          
        
      
        
          
            
              
                La Val, Taylor P.,
              
            
          
        
      
        
          
            
              
                Lund, Craig,
              
            
          
        
      
        
          
            
              
                Mullen, Madeline,
              
            
          
        
      
        
          
            
              
                Olsen, John,
              
            
          
        
      
        
          
            
              
                Sztain, Terra A.,
              
            
          
        
      
        
          
            
              
                Yoo, Jennifer,
              
            
          
        
      
        
          
            
              
                Moss, John A.,
              
            
          
        
      
        
          
            
              
                and Baum, Marc M.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Journal of Microbiological Methods</em>
    
    
      2014
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://www.sciencedirect.com/science/article/pii/S0167701214001055?via%3Dihub" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Asphalts, biochemically degraded oil, contain persistent, water-soluble compounds that pose a significant challenge to the isolation of PCR quality DNA. The adaptation of existing DNA purification protocols and commercial kits proved unsuccessful at overcoming this hurdle. Treatment of aqueous asphalt extracts with a polyamide resin afforded genomic microbial DNA templates that could readily be amplified by PCR. Physicochemically distinct asphalt samples from five natural oil seeps successfully generated the expected 291bp amplicons targeting a region of the 16S rRNA gene, illustrating the robustness of the method. DNA recovery yields were in the 50–80% range depending on how the asphalt sample was seeded with exogenous DNA. The scope of the new method was expanded to include soil with high humic acid content. DNA from soil samples spiked with a range of humic acid concentrations was extracted with a commercial kit followed by treatment with the polyamide resin. The additional step significantly improved the purity of the DNA templates, especially at high humic acid concentrations, based on qPCR analysis of the bacterial 16S rRNA genes. The new method has the advantages of being inexpensive, simple, and rapid and should provide a valuable addition to protocols in the field of petroleum and soil microbiology.</p>
  </span>
  
</div>

</li></ol>

<h4 class="pubyear"><i>Not peer reviewed</i></h4>

<h4 class="year">2014</h4>
<ol class="bibliography"><li>

<div id="holland2014draft">
  
    <span class="title">Draft genome sequence of the pyridinediol-fermenting bacterium <i>Synergistes jonesii</i> 78-1</span>
    <span class="author">
      
        
          
            
              
                Holland-Moritz, Hannah E,
              
            
          
        
      
        
          
            
              
                Coil, David A,
              
            
          
        
      
        
          
            
              
                Badger, Jonathan H,
              
            
          
        
      
        
          
            
              
                Dmitrov, George I,
              
            
          
        
      
        
          
            
              
                Khouri, Hoda,
              
            
          
        
      
        
          
            
              
                Ward, Naomi L,
              
            
          
        
      
        
          
            
              
                Robb, Frank T,
              
            
          
        
      
        
          
            
              
                and Eisen, Jonathan A
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Announcements</em>
    
    
      2014
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mra.asm.org/content/2/4/e00833-14" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mra.asm.org/content/ga/2/4/e00833-14.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Here we present the draft genome of Synergistes jonesii 78-1, ATCC 49833, a member of the Synergistes phylum. This organism was isolated from the rumen of a Hawaiian goat and ferments pyridinediols. The assembly contains 2,747,397 bp in 61 contigs.</p>
  </span>
  
</div>

</li></ol>

<h4 class="year">2013</h4>
<ol class="bibliography"><li>

<div id="holland2013draft">
  
    <span class="title">Draft genome sequence of Leucobacter sp. strain UCD-THU (phylum <i>Actinobacteria</i>)</span>
    <span class="author">
      
        
          
            
              
                Holland-Moritz, Hannah E,
              
            
          
        
      
        
          
            
              
                Bevans, Dakota R,
              
            
          
        
      
        
          
            
              
                Lang, Jenna M,
              
            
          
        
      
        
          
            
              
                Darling, Aaron E,
              
            
          
        
      
        
          
            
              
                Eisen, Jonathan A,
              
            
          
        
      
        
          
            
              
                and Coil, David A
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Announcements</em>
    
    
      2013
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
  
    [<a href="https://mra.asm.org/content/1/3/e00325-13" target="_blank">HTML</a>]
  
  
  <!-- use "pdflink" rather than pdf if you don't personally possess pdf" -->
  
    [<a href="https://mra.asm.org/content/ga/1/3/e00325-13.full.pdf" target="_blank">PDF</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Here we present the draft genome of Leucobacter sp. strain UCD-THU. The genome contains 3,317,267 bp in 11 scaffolds. This strain was isolated from a residential toilet as part of an undergraduate project to sequence reference genomes of microbes from the built environment.</p>
  </span>
  
</div>

</li></ol>


  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper">
    &copy; Copyright 2021 Hannah Holland-Moritz.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme.  Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>. </br> Photos not taken by the author or otherwise credited are from <a href="https://unsplash.com" target="_blank">Unsplash</a>. </br>

    
        Page updated: 2021-01-25.
    
  </div>

</footer>




    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


<!-- Load KaTeX -->
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.js"></script>
<script src="/assets/js/katex.js"></script>




<!-- Include custom icon fonts -->
<link rel="stylesheet" href="/assets/css/fontawesome-all.min.css">
<link rel="stylesheet" href="/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXXXXXX', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
